File: WMS.EntityCore/Pedido/clsBeCliente.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMS.EntityCore.Cliente
{
    public class clsBeCliente : ICloneable
    {
        [Column("IdCliente")]
        [DisplayName("IdCliente")]
        public int IdCliente { get; set; } = 0;

        [Column("IdEmpresa")]
        [DisplayName("IdEmpresa")]
        public int IdEmpresa { get; set; } = 0;

        [Column("IdPropietario")]
        [DisplayName("IdPropietario")]
        public int IdPropietario { get; set; } = 0;

        [Column("IdTipoCliente")]
        [DisplayName("IdTipoCliente")]
        public int IdTipoCliente { get; set; } = 0;

        [Column("IdUbicacionManufactura")]
        [DisplayName("IdUbicacionManufactura")]
        public int IdUbicacionManufactura { get; set; } = 0;

        [Column("codigo")]
        [DisplayName("codigo")]
        public string Codigo { get; set; } = "";

        [Column("nombre_comercial")]
        [DisplayName("nombre_comercial")]
        public string Nombre_comercial { get; set; } = "";

        [Column("nombre_contacto")]
        [DisplayName("nombre_contacto")]
        public string Nombre_contacto { get; set; } = "";

        [Column("telefono")]
        [DisplayName("telefono")]
        public string Telefono { get; set; } = "";

        [Column("nit")]
        [DisplayName("nit")]
        public string Nit { get; set; } = "";

        [Column("direccion")]
        [DisplayName("direccion")]
        public string Direccion { get; set; } = "";

        [Column("correo_electronico")]
        [DisplayName("correo_electronico")]
        public string Correo_electronico { get; set; } = "";

        [Column("activo")]
        [DisplayName("activo")]
        public bool Activo { get; set; } = false;

        [Column("realiza_manufactura")]
        [DisplayName("realiza_manufactura")]
        public bool Realiza_manufactura { get; set; } = false;

        [Column("user_agr")]
        [DisplayName("user_agr")]
        public string User_agr { get; set; } = "";

        [Column("fec_agr")]
        [DisplayName("fec_agr")]
        public DateTime Fec_agr { get; set; } = DateTime.Now;

        [Column("user_mod")]
        [DisplayName("user_mod")]
        public string User_mod { get; set; } = "";

        [Column("fec_mod")]
        [DisplayName("fec_mod")]
        public DateTime Fec_mod { get; set; } = DateTime.Now;

        [Column("despachar_lotes_completos")]
        [DisplayName("despachar_lotes_completos")]
        public bool Despachar_lotes_completos { get; set; } = false;

        [Column("sistema")]
        [DisplayName("sistema")]
        public bool Sistema { get; set; } = false;

        [Column("es_bodega_recepcion")]
        [DisplayName("es_bodega_recepcion")]
        public bool Es_bodega_recepcion { get; set; } = false;

        [Column("es_bodega_traslado")]
        [DisplayName("es_bodega_traslado")]
        public bool Es_bodega_traslado { get; set; } = false;

        [Column("idubicacionvirtual")]
        [DisplayName("idubicacionvirtual")]
        public int Idubicacionvirtual { get; set; } = 0;

        [Column("referencia")]
        [DisplayName("referencia")]
        public string Referencia { get; set; } = "";

        [Column("control_ultimo_lote")]
        [DisplayName("control_ultimo_lote")]
        public bool Control_ultimo_lote { get; set; } = false;

        [Column("control_calidad")]
        [DisplayName("control_calidad")]
        public bool Control_calidad { get; set; } = false;

        [Column("IdUbicacionAbastecerCon")]
        [DisplayName("IdUbicacionAbastecerCon")]
        public int IdUbicacionAbastecerCon { get; set; } = 0;

        [Column("IdBodegaAreaSAP")]
        [DisplayName("IdBodegaAreaSAP")]
        public int IdBodegaAreaSAP { get; set; } = 0;

        [Column("es_proveedor")]
        [DisplayName("es_proveedor")]
        public bool Es_proveedor { get; set; } = false;

        public clsBeCliente() { }

        public object Clone()
        {
            return MemberwiseClone();
        }
    }
}

File: TOMIMSV4/Entity/Mantenimientos/Cliente/clsBeCliente.vb
Public Class clsBeCliente
    Implements ICloneable
    Public Property IdCliente() As Integer = 0
    Public Property IdEmpresa() As Integer = 0
    Public Property IdPropietario() As Integer = 0
    Public Property IdTipoCliente() As Integer = 0
    Public Property IdUbicacionManufactura() As Integer = 0
    Public Property Codigo() As String = ""
    Public Property Nombre_comercial() As String = ""
    Public Property Nombre_contacto() As String = ""
    Public Property Telefono() As String = ""
    Public Property Nit() As String = ""
    Public Property Direccion() As String = ""
    Public Property Correo_electronico() As String = ""
    Public Property Activo() As Boolean = False
    Public Property Realiza_manufactura() As Boolean = False
    Public Property User_agr() As String = ""
    Public Property Fec_agr() As Date = Date.Now
    Public Property User_mod() As String = ""
    Public Property Fec_mod() As Date = Date.Now
    Public Property Despachar_lotes_completos() As Boolean = False
    Public Property Sistema() As Boolean = False
    Public Property Es_bodega_recepcion() As Boolean = False
    Public Property Es_Bodega_Traslado() As Boolean = False
    Public Property IdUbicacionVirtual() As Integer = 0
    Public Property Control_Ultimo_Lote As Boolean = False
    Public Property Referencia As String = ""
    Public Property Control_Calidad As Boolean = False
    Public Property IdUbicacionAbastecerCon As Integer = 0
    Public Property IdBodegaAreaSAP As Integer = 0
    Public Property Es_Proveedor As Boolean = False
    Public Property Codigo_Empresa_ERP As String = ""
    Public Property IdProductoEstadoDefecto As Integer = 0

    Sub New()
    End Sub
    Sub New(ByRef IdCliente As Integer, ByVal IdEmpresa As Integer, ByVal IdPropietario As Integer, ByVal IdTipoCliente As Integer, ByVal IdUbicacionManufactura As Integer, ByVal codigo As String, ByVal nombre_comercial As String, ByVal nombre_contacto As String, ByVal telefono As String, ByVal nit As String, ByVal direccion As String, ByVal correo_electronico As String, ByVal activo As Boolean, ByVal realiza_manufactura As Boolean, ByVal user_agr As String, ByVal fec_agr As Date, ByVal user_mod As String, ByVal fec_mod As Date, ByVal despachar_lotes_completos As Boolean, ByVal sistema As Boolean)
        Me.IdCliente = IdCliente
        Me.IdEmpresa = IdEmpresa
        Me.IdPropietario = IdPropietario
        Me.IdTipoCliente = IdTipoCliente
        Me.IdUbicacionManufactura = IdUbicacionManufactura
        Me.Codigo = codigo
        Me.Nombre_comercial = nombre_comercial
        Me.Nombre_contacto = nombre_contacto
        Me.Telefono = telefono
        Me.Nit = nit
        Me.Direccion = direccion
        Me.Correo_electronico = correo_electronico
        Me.Activo = activo
        Me.Realiza_manufactura = realiza_manufactura
        Me.User_agr = user_agr
        Me.Fec_agr = fec_agr
        Me.User_mod = user_mod
        Me.Fec_mod = fec_mod
        Me.Despachar_lotes_completos = despachar_lotes_completos
        Me.Sistema = sistema
    End Sub
    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class
